tests/functional/locators.py

```diff
@@ -9,7 +9,7 @@ class LoginPageLocators:
 
 
 class NavigationLocators:
-    LOG_OUT_LINK = (By.LINK_TEXT, "Sign out")
+    LOG_OUT_LINK = (By.NAME, "logout")
```

Comment:
Can we continue to locate it via the visible interface text here?  Might need some xpath like `button[text()='Sign out']`?